src/openforms/registrations/contrib/objects_api/registration_variables.py

```diff
@@ -1,14 +1,17 @@
 from __future__ import annotations
 
+from datetime import datetime
 from typing import TYPE_CHECKING
 
 from django.utils.translation import gettext_lazy as _
 
+from openforms.authentication.constants import AuthAttribute
```

Comment:
you're importing from a module other than the `service.py` module, this is not public API. The same goes for the `FormAuth` type - I think you're abusing these types.

co-sign also doesn't have any information about `machtigen` or `loa` afaik, so I'd really define a new typed dict for this instead. I would expect something already exists because of the shape of co sign v1 and v2?

Possibly you can make `BaseAuth` TypedDict public for this, as that captures the co sign v2 correctly.

Reply:
Makes sense for `FormAuth`, however for `AuthAttribute` I think it makes sense to consider it "public" (and maybe in general the `constants` modules from each app).

`AuthAttribute` is imported 16 times in the code base (without taking tests into account).

Reply:
yeah I'm trying to get bad practices removed :)

you can export the public constants from the service module

Reply:
Will update other usage in another PR